src/pytest_plugins/filler/filler.py

```diff
@@ -1038,6 +1063,22 @@ def pytest_generate_tests(metafunc: pytest.Metafunc):
                 indirect=True,
             )
 
+            # Handle gas benchmark values parametrization
+            if "gas_benchmark_value" in metafunc.fixturenames:
+                gas_benchmark_values = metafunc.config.getoption("gas_benchmark_value")
+                if gas_benchmark_values:
+                    gas_values = [int(x) for x in gas_benchmark_values.replace(" ", "").split(",")]
```

Comment:
```suggestion
                    gas_values = [int(x.strip()) for x in gas_benchmark_values.split(",")]
```
Not sure if necessary but maybe in case there's a white space between commas?

Reply:
It is necessary to handle cases like `5, 10,   20,30`, string with white spaces between. But the current `.replace(" ", "")` method already removes the white space